src/app/products/products-routing.module.ts

```diff
@@ -1,43 +1,49 @@
-/** Angular Imports */

```

Comment:
Same lines have been removed and added here again, please ensure only the required new additions are present in the PR.

Reply:
@abhaychawla I'm not sure what the issue is here on my end. It shows everything as changed and I can't get it to just show specific changes.

Reply:
Let me try something